.github/workflows/sync-docs-from-node.yml

```
@@ -26,878 +26,342 @@ on:
 
-            # Remove files that no longer exist in source
-            DELETED=0
-            for dest_file in "${EXISTING_FILES[@]}"; do
-              if [ -f "$dest_file" ]; then
-                rm "$dest_file"
-                printf -- "- Deleted: \`%s\`\n" "$(basename "$dest_file")" >> $SYNC_REPORT
-                DELETED=$((DELETED + 1))
-              fi
-            done
+      - name: Sync files based on matrix type
+        id: sync_${{ matrix.sync_type }}
```

actionlint (1.7.7)

context "matrix" is not allowed here. no context is available here. see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)